### PR TITLE
refactor to introduce InputProcessor;change text_plain to 10 char fileset

### PR DESCRIPTION
**Pull request by bradflood**

### CODE REVIEW

File: load/InputFileset.py [hunk 1]
```diff
@@ -385,12 +346,14 @@ def batchName(self):
 
 
 if (__name__ == '__main__'):
+	from InputProcessor import *	
+
 	config = Config()
 	s3Client = AWSSession.shared().s3Client
```
Comment (vichugofsl) [inline load/InputFileset.py:352]: I think this line can be removed. The same for the lines 353 and 354

**Review by vichugofsl — CHANGES_REQUESTED** [review]
I think that we can remove the line 352 from the InputFileset.py file:

https://github.com/faithcomesbyhearing/dbp-etl/pull/17/files#r847551285

Also, we can remove the line: 83 from InputProcessor.py file:

https://github.com/faithcomesbyhearing/dbp-etl/pull/17/files#r847553537

File: load/InputFileset.py (hunk 1 above)
Comment (bradflood) [reply load/InputFileset.py:352]: I get an error if I remove it

File: load/InputProcessor.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		if self.locationType == InputFileset.LOCAL:
+			return self.location + os.sep + self.filesetPath
+		else:
+			return self.location + "/" + self.filesetPath
+
+
+
+if (__name__ == '__main__'):
+	config = Config()
+	s3Client = AWSSession.shared().s3Client
```
[the remark this answers is not in the document]
Comment (bradflood) [reply load/InputProcessor.py]: removed line 83
s3Client = AWSSession.shared().s3Client

**Review by vichugofsl — APPROVED** [review]

**Comment by vichugofsl** [comment]
ok @bradflood , it looks good fo me

**Pull request merged**